### DISCUSSION TITLE
Examples of Lwarp in use

### DISCUSSION THREAD

**Discussion by bdtc** [bdtc/lwarp#19] [Show and tell]
Some examples of **lwarp** in use.  Feel free to add more to this thread.

* [ The TikZ and PGF Packages](https://tikz.dev/) (**lwarp** with additional post-processing.) (Till Tantau, et al.)
* [Mongolian (Cyrillic)](https://mongol-surah.github.io/cyrillic/cyrillic-grammar/index.html) (清格尔泰   著)
* [Probability with Measure](https://nicfreeman1209.github.io/MAS350_lwarp_example/MAS350/out_html/MAS350-converted.html) (Dr Nic Freeman)
* [A New Maximal Subgroup of E8 in Characteristic 3](https://web.mat.bham.ac.uk/D.A.Craven/docs/papers/html/newmaxinE82103.html)  (David A. Craven, David I. Stewart and Adam R. Thomas)
* [MA20216: Algebra 2A](https://people.bath.ac.uk/feb/ma20216/notes/) (Fran Burstall)
* [MA30061 Control Theory, 2023–2024](https://people.bath.ac.uk/mo221/MA30061.bho/notes/index.html) (Mark Opmeer)

**Comment by SwitWu** [comment]
+ [Beamer HTML manual](https://www.beamer.plus)

**Comment by bdtc** [comment]
[Sample HTML file: produced with lwarp, with mathematical formulae displayed with MathJax, and xymatrix commutative diagrams displayed with XyJax-v3](https://joaofariamartins.github.io/Lwarp-JFM/Sample_Lwarp_Xy-jax-split-index.html)

**Comment by Rmano** [comment]
I just managed to convert the[ `circuitikz` manual](https://rmano.github.io/circuitikz/index.html). A bit of bumps in the path, but I like the final result (it's not so beautiful as TikZ and pgfplots are, but this is just `lwarp`, no additional passes.

**Reply by asmwarrior** [reply]
> I just managed to convert the[ `circuitikz` manual](https://rmano.github.io/circuitikz/index.html). A bit of bumps in the path, but I like the final result (it's not so beautiful as TikZ and pgfplots are, but this is just `lwarp`, no additional passes.

Very nice document, good work!

**Comment by asmwarrior** [comment]
I see one issue when I view the page: https://rmano.github.io/circuitikz/node-Using-bipoles-in-circuits.html

When I click an item of the TOC in the left side panel, I see some times, the TOC will scroll to the beginning, do you guys see this issue?

**Reply by Rmano** [reply]
Yes, I noticed that sometimes. Probably a bug in the javascript... You can see the source for the page here: https://github.com/Rmano/circuitikz/tree/html/lwarpfiles

**Reply by asmwarrior** [reply]
> Yes, I noticed that sometimes. Probably a bug in the javascript... You can see the source for the page here: https://github.com/Rmano/circuitikz/tree/html/lwarpfiles

Hopefully this issue will be fixed.  I'm not sure whether it is inside your own files or in the lwarp package.

**Reply by bdtc** [reply]
Lwarp is almost Javascript-free.  The sideTOC is not using any Javascript at all.  A copy is added to each web page, without any reference of "I am here.", which explains why it starts at the top again.  I wanted it to work even in contexts where Javascript is not available, and I am not a Javascript developer.

There is much room for improvement, including the time it takes to recompile each page's sideTOC, the lack of drop-down menu folding, highlighting the current position, and so on, but all of this will be a "some year" thing because I am pretty busy with other things right now.  Sorry!

Adapting to the tagging project's work will probably be the next big task.  Many changes required.

**Reply by asmwarrior** [reply]
> Lwarp is almost Javascript-free. The sideTOC is not using any Javascript at all. A copy is added to each web page, without any reference of "I am here.", which explains why it starts at the top again. I wanted it to work even in contexts where Javascript is not available, and I am not a Javascript developer.
> 
> There is much room for improvement, including the time it takes to recompile each page's sideTOC, the lack of drop-down menu folding, highlighting the current position, and so on, but all of this will be a "some year" thing because I am pretty busy with other things right now. Sorry!
> 
> Adapting to the tagging project's work will probably be the next big task. Many changes required.

Thanks for the explanation. So it looks like without having some Javascript code, it is very hard to solve this issue.
